orchestra/utils/common_helpers.py

```diff
@@ -33,11 +34,17 @@ def get_relevance_and_completion_changes(old_todo, new_todo):
     # When activity_log is updated, `todo_change = ''`
     # to avoid triggering any slack messages
     todo_change = ''
-    if old_todo.completed != new_todo.completed:
-        todo_change = 'complete' if new_todo.completed else 'incomplete'
-    elif old_todo.skipped_datetime != new_todo.skipped_datetime:
-        todo_change = 'not relevant' \
-            if new_todo.skipped_datetime else 'relevant'
+    if new_todo.status != old_todo.status:
+        if new_todo.status == Todo.Status.COMPLETED.value and \
```

Comment:
should any of the stuff below be `elif` to be more explicit?

Reply:
Fixed